dbms/src/Server/Server.cpp

```diff
@@ -1039,12 +1056,35 @@ int Server::main(const std::vector<std::string> & /*args*/)
         LOG_INFO(log, "UniPS is not enabled for proxy, page_version={}", STORAGE_FORMAT_CURRENT.page);
     }
 
+    std::visit(
```

Comment:
We'd better put thses args parsing into `TiFlashProxyConfig`.
PTAL at this PR: https://github.com/CalvinNeo/tiflash/pull/12